libs/auth/src/cac/common_access_card.ts

```diff
@@ -412,8 +413,23 @@ export class CommonAccessCard implements CommonAccessCardCompatibleCard {
       return Buffer.of();
     }
 
-    const certTlv = data.subarray(0, -5); // Trim metadata
-    const [, , certInDerFormat] = parseTlv(PUT_DATA.CERT_TAG, certTlv);
+    const [certInDerFormatRemainder, { value: certInDerFormat }] =
+      parseTlvPartial(PUT_DATA.CERT_TAG, data);
+    const [certInfoRemainder, { value: certInfo }] = parseTlvPartial(
+      PUT_DATA.CERT_INFO_TAG,
+      certInDerFormatRemainder
+    );
+    // NOTE: CACs seem to return 0x01 for this. I'm not sure what it means, but
+    // it's not compressed.
+    assert(certInfo[0] === 0x01, 'Expected cert info to be uncompressed');
```

Comment:
Interesting! According to the page 35 of the [NIST PIV standard](https://nvlpubs.nist.gov/nistpubs/specialpublications/nist.sp.800-73-4.pdf):

> Thus, for a certificate encoded in uncompressed form CertInfo shall be
> 0x00, and for a certificate encoded using GZIP compression CertInfo shall be 0x01.

Odd that it's not actually compressed

Reply:
ae2551ddbb097efe0eb4dcc19f1e8540af4797d2